pkg/policy/policy.go

```diff
@@ -63,6 +66,26 @@ func (p Policy) TotalQueries() int {
 	return count + len(p.Checks)
 }
 
+func (p Policy) Filter(path string) Policy {
```

Comment:
I feel like this can also be used in the executor to find a policy / check, wdyt?

Reply:
I too think all filtering should be unified so that if (erm, when) it's buggy, it's buggy the same way 😂

Reply:
Added support for filtering checks